packages/terra-functional-testing/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * Added
   * Added `ignoreScreenshotMismatch` flag to disable test failure on snapshot mismatch.
   * Added configuration to retrieve and unzip screenshots from registry.
+  * Added FileOutputReporter reporter that logs wdio test output to separate files based on locale, theme, and form-factor
```

Comment:
This got added to the released section.

Reply:
https://github.com/cerner/terra-toolkit/pull/665/files#diff-5121be21da689ee0a1bd5a529bbd578dc2bde92ce93b497f118e58290be63dc1R8-R10